internal/controller/constants.go

```diff
@@ -27,14 +29,20 @@ const (
 	// Image of the OLS application server
 	// todo: image vesion should synchronize with the release version of the lightspeed-service-api image.
 	OLSAppServerImageDefault = "quay.io/openshift/lightspeed-service-api:latest"
+	// Image of the OLS application redis server
+	OLSAppRedisServerImageDefault = "quay.io/vchalla/redis:latest"
```

Comment:
This is used just for testing purposes. Will need to me changed before we are good with merging this PR.